fix incorrect operator lookup for postincrement (#4536)
cython/cython: docs/src/userguide/migrating_to_cy30.rst

annotations differs from Cython's, Cython 3 now supports setting the
``annotation_typing`` :ref:`directive <compiler-directives>` on a
per-class or per-function level.

C++ postincrement/postdecrement operator
========================================

> Cython 3 differentiates between pre/post-increment and pre/pos-decrement

Reviewer: ```suggestion
Cython 3 differentiates between pre/post-increment and pre/post-decrement
```

> operators (Cython 0.29 implemented both as predecrement operator).

Reviewer: ```suggestion
operators (Cython 0.29 implemented both as pre(in/de)crement operator).
```

Not sure if this phrasing is clearer but it kind of implied that Cython 0.29 subtracted one whatever you did.

> This only has an effect when using ``operator.postdecrement`` / ``operator.postincrement``.

Reviewer: ```suggestion
This only has an effect when using ``cython.operator.postdecrement`` / ``cython.operator.postincrement``.
```

?
